tests/feature/services/ShopService/ShopifyTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+
+namespace Tychovbh\Tests\Mvc\feature\services\ShopService;
+
+
+use GuzzleHttp\Client;
+use Illuminate\Support\Arr;
+use Tychovbh\Mvc\Services\ShopService\Shopify;
+use Tychovbh\Tests\Mvc\TestCase;
+
+class ShopifyTest extends TestCase
+{
+    private function shopifyService()
+    {
+        $guzzle = new Client();
+        return new Shopify($guzzle);
+    }
+
+    /**
+     * @test
+     */
+    public function itCanIndexProducts()
+    {
+        $products = $this->shopifyService()->products();
+        $this->assertTrue(Arr::has($products, '0.id'));
+    }
+
+    /**
+     * @test
+     */
+    public function itCanShowProduct()
+    {
+        $id = 5979585118361;
```

Comment:
this ID may not exists one day, make this method depends on itCanIndex and than do a request with the first record